data/systemd/snapd.core-fixup.service.in

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Automatically repair incorrect owner/permissons on core devices
+Before=snapd.service
+ConditionPathExists=/writable/system-data
+ConditionPathExists=!/var/lib/snapd/device/ownership-change.after
+Documentation=man:snap(1)
+
+[Service]
+Type=oneshot
+ExecStart=@libexecdir@/snapd/snapd.core-fixup.sh
+RemainAfterExit=true
```

Comment:
I've read the doc on this flag but it's not clear to me why we don't want the default here ("false")?